qubit-sdk/src/main/java/com/qubit/android/sdk/internal/placement/interactor/PlacementInteractorImpl.kt

```diff
@@ -68,6 +101,9 @@ internal class PlacementInteractorImpl(
             placementContent.callbacks.clickthrough
         )
         val placement = PlacementImpl(placementContent.content, callbackConnector)
+        if (shouldUpdateCache) {
+          placementRepository.save(cacheKey, placementModel)
+        }
         onSuccess(placement)
       } catch (e: Exception) {
```

Comment:
What kind of errors can be caught here?

Reply:
Actually it seems that none. Removed try..catch